packages/embed/src/factories/create-popover/create-popover.spec.ts

```diff
@@ -16,12 +15,14 @@ afterEach(() => {
 describe('#createSidetab', () => {
   describe('#open', () => {
     it('should open', () => {
+      popover = createPopover('formId')
```

Comment:
You can use `beforeEach` to recreate this before each test.

Same for sidetab tests.

Reply:
it was like that, and I changed it because sometimes I need to pass different `options` depending on the test, so I had to `unmount` and `remount` the embed with the new options for those tests, which I think is not correct

Reply:
There already is `afterEach` that does `popover.unmount()`. If you dont want to do unmount in specific test you can restructure describes and have the `beforeEach` only for those where you need it.

```
describe('default popover', () => {
  beforeEach(() => {
    popover = createPopover('formId')
  })
  it('xxx', () => {
    popover.open()
    // ...
  })
})

describe('different popover', () => {
  it('xxx', () => {
    createPopover('formId', { buttonColor: white })
    // ...
  })
})
```

Reply:
hmmm great idea, on it